bindings/java/src/org/sleuthkit/datamodel/DataSource.java

```diff
@@ -76,16 +76,16 @@
 	void setAcquisitionDetails(String details) throws TskCoreException;
 
 	/**
-	 * Sets the acquisition settings used as well as module name and module version.
-	 * 
-	 * @param acquisitionSettings Any settings specific to the acquisition. May
-	 *                            be Null.
-	 * @param moduleName          The module name. May be Null
-	 * @param moduleVersion       The module's version number. May be Null.
+	 * Sets the acquisition tool details such as its name, version number and 
+	 * any settings used during the acquisition tool to acquire data.
```

Comment:
Quick comment, this sentence should probably be: "Sets the acquisition tool details such as its name, version number and any settings used during the acquisition to acquire data."